### DISCUSSION TITLE
STM32MP157C_DK2 board openocd.cfg file empty

### DISCUSSION THREAD

**Discussion by juliaoconnell-eaton** [zephyrproject-rtos/zephyr#45115] [RFC]
Hi there,

I'm trying to set-up OpenOCD debugging for the STM32MP157C_DK2 board in Eclipse IDE. Upon investigation the file [boards/arm/stm32mp157c_dk2/support/openocd.cfg](https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/arm/stm32mp157c_dk2/support/openocd.cfg) is empty. 

Based on an example from the [Nucleo-F492ZI board](https://github.com/zephyrproject-rtos/zephyr/blob/d92e3b88bacc6e6fc1b6222421ef4accdf2d574e/boards/arm/nucleo_f429zi/support/openocd.cfg) and the [OpenOCD target/stm32mp15x.cfg](https://github.com/zephyrproject-rtos/openocd/blob/zephyr-20210630/tcl/target/stm32mp15x.cfg) the config file should look something like this:
```
source [find board/stm32mp15x_dk2.cfg]

$_CHIPNAME.cpu0 configure -event gdb-attach {
        echo "Debugger attaching: halting execution"
        reset halt
        gdb_breakpoint_override hard
}

$_CHIPNAME.cpu0 configure -event gdb-detach {
        echo "Debugger detaching: resuming execution"
        resume
}
```
Thanks!

**Comment by erwango** [comment]
Zephyr implementation on STM32MP1 is for now limited to Cortex-M4 core, using Cortex-A as Linux host.
Instructions for flashing and debug are provided here: 
https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/arm/stm32mp157c_dk2/doc/stm32mp157_dk2.rst#debugging

This is why this differs from usual STM32 targets
